cli/lib/exec/open.js

```diff
@@ -67,7 +69,17 @@ const processOpenOptions = (options = {}) => {
 module.exports = {
   processOpenOptions,
   start (options = {}) {
-    const args = processOpenOptions(options)
+    let args
+
+    try {
+      args = processOpenOptions(options)
+    } catch (err) {
+      if (err.details) {
+        return exitWithError(err.details)()
+      }
+
+      throw err
+    }
```

Comment:
Seems `args` should be scoped to `open`, the only place it's used?

```js
function open () {
  try {
    const args = processOpenOptions(options)
    return spawn.start(args, {
      dev: options.dev,
      detached: Boolean(options.detached),
      stdio: 'inherit',
    })
  } catch (err) {
    if (err.details) {
      return exitWithError(err.details)()
    }
    throw err
  }
}
```

Optional, but I feel this is an improvement.

Reply:
I also made the same change to `run` so they are symmetrical.